website/docs/reference/api/legacy/unleash/admin/features-v2.md

```diff
@@ -696,8 +703,15 @@ Content-Type: application/json; charset=utf-8
 ```
 
 ### PATCH variants for a feature toggle
+:::caution
+
+From 4.21 variants are tied to an environment. Check our Open API docs for the up-to-date docs: https://docs.getunleash.io/reference/api/unleash/features
```

Comment:
I'm not sure what the link does here? What's the purpose of it? Why are you linking them to that page exactly?

```suggestion
Since Unleash 4.21, variants are tied to environments.
```

Reply:
Sorry, I'm not sure I understand. Why are you linking them there? Are you saying that the OpenAPI docs you're pointing to are the more updated versions of the doc for this specific endpoint?

Reply:
Isn't this documentation "legacy"? We've found some payloads which are not up-to-date (due to a non-intentional change), but OpenApi should have the correct shapes for these API endpoints. This would avoid maintaining the same documentation in two places... 

But we can talk about it, I'm sensing there's something I'm missing here

Reply:
Yes, it is. I don't think you're missing anything. This is legacy documentation and shouldn't really be updated (though I like the idea of adding links to the new docs). So yes, this is a good change. I just wanted to understand where you want to link them, but I think I got it! 😄